.github/workflows/cloud-regress.yml

```diff
@@ -73,15 +79,42 @@ jobs:
           path: /tmp/neon/
           prefix: latest
 
+      - name: Create a new branch
+        id: create-branch
+        uses: ./.github/actions/neon-branch-create
+        with:
+          api_key: ${{ secrets.NEON_STAGING_API_KEY }}
+          project_id: ${{ vars[format('PGREGRESS_PG{0}_PROJECT_ID', matrix.pg-version)] }}
```

Comment:
I liked one of the previous approaches with `${{ vars.PGREGRESS_PG16_PROJECT_ID }}` listed in matrix. I think the readability was better then

Reply:
I agree about readability but in this case, the redundant project name is shown with the version.